internal/backend/grpcServer/forms/json_form.go

```diff
@@ -0,0 +1,83 @@
+package forms
+
+type ReferencePointForm struct {
+	Timestamp     TimeStruct       `json:"timestamp"`
+	CorrelationId string           `json:"correlationId"`
+	Operation     string           `json:"operation"`
+	Operands      []OperandsStruct `json:"operands"`
+	Error         ErrorStruct      `json:"error"`
+}
+
+type RssiForm struct {
+	Readings []ReadingStruct `json:"readings"`
+}
+
+type XYZForm struct {
+	X           float32             `json:"x"`
+	Y           float32             `json:"y"`
+	Z           float32             `json:"z"`
+	Accuracy    int                 `json:"accuracy"`
+	Tag_id      string              `json:"tag_id"`
+	Chain_delay []Chain_delayStruct `json:"chain_delay"`
+}
+type TimeStruct struct {
+	Seconds int `json:"seconds"`
+	Nanos   int `json:"nanos"`
+}
+
+type OperandsStruct struct {
```

Comment:
Maybe just `Operands` instead of `OperandsStruct` since struct is defined right afterwards